demos/gesture_recognition_demo/python/gesture_recognition_demo.py

```diff
@@ -251,4 +252,5 @@ def main():
 
 
 if __name__ == '__main__':
+    multiprocessing.set_start_method('spawn')  # https://github.com/opencv/opencv/issues/5150#issuecomment-197413178 and https://github.com/opencv/opencv/issues/5150#issuecomment-197540235
```

Comment:
```suggestion
    # https://github.com/opencv/opencv/issues/5150#issuecomment-197413178
    # https://github.com/opencv/opencv/issues/5150#issuecomment-197540235
    multiprocessing.set_start_method('spawn')
```

Keep the line lengths reasonable.